### DISCUSSION TITLE
Questions Regarding Figure 8 in Your Article "The Parcels v2.0 Lagrangian Framework"

### DISCUSSION THREAD

**Discussion by Caiyuan6** [Parcels-code/Parcels#1598] [Q&A]
Hi, @erikvansebille.
I am currently reading your article titled "**The Parcels v2.0 Lagrangian framework: new field interpolation schemes**"[(https://gmd.copernicus.org/articles/12/3571/2019/)](https://gmd.copernicus.org/articles/12/3571/2019/) and attempting to follow the methods and content to perform graphical analysis. However, I have some questions regarding Figure 8. Specifically, I am unclear about the significance of Figure 8 and how to analyze it. Additionally, I would like to understand the significance of analyzing particles released at different times and their ages.

As a beginner in the field of physical oceanography, I would greatly appreciate your guidance. Could you please explain the specific meaning of Figure 8 in detail and provide some guidance on how to analyze this figure? Furthermore, could you please explain in detail the significance of analyzing particles released at different times and their ages? What role does this play in the research, and what important information does it provide?

Thank you very much for your assistance!

Sincerely,

Cyuan

**Comment by erikvansebille** [comment]
Dear @Cyuan-hub, I'm not sure that the Parcels GitHub is the best place to ask questions about the content of papers and physical oceanography; I'd prefer to keep this channel for technical questions. on the use of parcels. So I'm closing this discussion and if you want you can email me
